rust/ql/consistency-queries/VariableConsistency.ql

```diff
@@ -0,0 +1,8 @@
+/**
+ * @name Variable inconsistencies
+ * @description Lists the variable inconsistencies in the database.  This query is intended for internal use.
+ * @kind table
+ * @id rust/diagnostics/variable-consistency
+ */
+
+import codeql.rust.internal.VariableConsistency
```

Comment:
Do you want a summary for this query adding to DCA?  I have some work to do there anyway...

Reply:
I'll merge it into AST inconsistencies.